### DISCUSSION TITLE
#35 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by bobjiang** [tog-ETH-er/togETHer#10] [General]
Agenda:

- Community Updates
- CryptoTechNight proposal  https://github.com/tog-ETH-er/tog-eth-er.github.io/issues/8
- Fireside discussion

CryptoTechNight (9票全票通过)

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by Emilyxiannv** [comment]
### DAOSquare
翻译了几篇关于web3和DAO的文章，有兴趣可以阅读：
https://mp.weixin.qq.com/s/bTL6GF8DhlU0pzNGYywYxg
https://mp.weixin.qq.com/s/Fbt0Xwy3eVr3uJ0Y1GRccw
https://mp.weixin.qq.com/s/9WlhBUoejIRXlZJGDIgbXg

**Comment by 0x-stan** [comment]
# Dapp-Learning

近两周工作进展

## bounty

新建bounty板块

https://github.com/Dapp-Learning-DAO/Dapp-Learning/discussions/categories/bounty

## repo update

30个PR #632 ～ #663

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了4场分享会

- Curve  V1 StableSwap code review 
  - by 三火
  - https://youtu.be/pjwLIsZydmk
  - https://youtu.be/1BOilqC3x8U
- Gitcoin及Grant介绍
  - by Bob
  - https://youtu.be/pSQXaBjAGzI
- StarkNet 中文开发者 Meetup (转发)
  - by Leo & Henri
  - https://youtu.be/my2n-2rDmls
  - https://youtu.be/yn6T6aau8BI
- The Graph 体系架构与经济模型
  - by Jerry
  - https://youtu.be/Ki_cuT2oS04

 
## 翻译小组

- MIT公开课翻译第11集发布
  - https://mp.weixin.qq.com/s/dLM8o26GDOiv57jA3YtpXA
- MIT公开课翻译第12集审稿

**Comment by winkybupt** [comment]
## 原语里弄
1. Eigenphi 招募研究员，可以使用他们的底层平台开展 MEV 研究
2. 设计了 Lane’s feed，欢迎试用 https://www.notion.so/PrimitivesFeed-b1fc4a67ec42434fa447a921c699b74a

**Comment by bobjiang** [comment]
- https://greenpill.party/ 中文解读 
- https://open.spotify.com/show/0l6aXWC94dd0RA3tkKfxjd
- Gitcoin GR13 , Mar 24
- Gitcoin data sciece  https://gov.gitcoin.co/t/data-science-gitcoindao/10088

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网翻译继续进行中
- 准备贡献者春天周边的设计和制作

**Comment by xrdavies** [comment]
# Rebase Community

## Rostra
新的Sprint
https://github.com/rebase-network/rostra-app   
https://github.com/rebase-network/rostra-backend  

## CodingDay
3.26 开始举办 Online CodingDay
https://github.com/rebase-network/hello-world/discussions/81

## Rebase 大学
举办了一次分享

## Web3极客日报
正常更新14期

## Who-Is-Hiring
发布6期招聘信息
https://github.com/rebase-network/who-is-hiring/issues

## Event Pool
发布4期 Event
https://github.com/rebase-network/event-pool/issues

**Reply by bobjiang** [reply]
would you share the links for different summaries?

**Reply by xrdavies** [reply]
> would you share the links for different summaries?
Sure. Thank for hint

**Comment by stephanieeee-ho** [comment]
## EthereumCN
1. 常规翻译输出
2. 举办 StarkNet 中文开发者社区Meetup
   https://mp.weixin.qq.com/s/r7cNMWYPP-9sqjLcKz64Aw

**Comment by 0x68616f79756e** [comment]
ETHPlanet
1.筹备协调ETHShanghai 活动以及Side Event，市集等。
2.准备启动EDCON以及协调ETAC大会。
3.设计制作以太坊周边。
4.筹备Scholarship

**Comment by Jackieyewang** [comment]
BTCU

1. 协调各个学校的关系，组织委员会成立
2. oneblock一起做moonbeam入门
3. 与seedao组织关于“大学生要不要辍学all in crypro”的讨论
4. 科研社群dao成立
